src/core/export/export.service.js

```diff
@@ -23,5 +23,5 @@ function flattenObject(obj, separator = ', ') {
 }
 
 export {
-	flattenObject
+	flatView
```

Comment:
**export** service exposes single function called `flatView`. Consider to rename file to flatten or like this